codec-http/src/main/java/io/netty/handler/codec/http/multipart/HttpPostMultipartRequestDecoder.java

```diff
@@ -771,6 +771,18 @@ private InterfaceHttpData findMultipartDisposition() {
                                 throw new ErrorDataDecoderException(e);
                             }
                             currentFieldAttributes.put(HttpHeaderValues.CHARSET, attribute);
+                        } else if (contents[i].contains("=")) {
+                            String name = StringUtil.substringBefore(contents[i], '=');
+                            String values = StringUtil.substringAfter(contents[i], '=');
+                            Attribute attribute;
+                            try {
+                                attribute = factory.createAttribute(request, name, cleanString(values));
```

Comment:
The `values` may be a [quoted string](https://www.rfc-editor.org/rfc/rfc9110.html#name-quoted-strings) according to the [spec for parameters](https://www.rfc-editor.org/rfc/rfc9110.html#name-parameters), which `cleanString()` will not handle correctly.

We should instead use `cleanString()` for the `name`, which is always a token.

Reply:
Resolved.